components/properties/src/sets.rs

```diff
@@ -1835,10 +1835,13 @@ pub fn load_for_general_category_group(
     Ok(CodePointSetData::from_code_point_inversion_list(set))
 }
 
-/// Returns a type capable of looking up values for a property specified as a string, as long as it is an
-/// [ECMA262-subset binary property][ecma], using strict matching on the names in the spec.
+/// Returns a type capable of looking up values for a property specified as a string, as long as it is a
+/// [binary property listed in ECMA-262][ecma], using strict matching on the names in the spec.
 ///
-/// This handles every property required by ECMA262, except for:
+/// Returns an error when data fails to load, and returns `Ok(None)` when the property name does not match
+/// any of the ECMA-262 binary properties.
```

Comment:
Thought (optional): should it be `Option<Result>` instead of `Result<Option>`? It's kind-of weird to return `Ok` when it's not actually ok? I guess that makes it harder with FFI where the result needs to be on the outside

Reply:
Yeah that makes sense

Reply:
Actually this makes me think that having an error variant is the right way to go here (and I can document the error variant on the FFI function)